### PR TITLE
168066525 Users Should Be Able To Get The Landing Page Article Feed

### PR DESCRIPTION
**Pull request by kodek-sleuth**
#### What does this PR do?
Should enablea user to create and update an article
#### Description of Task to be completed
- Setup actions for making call to API
#### How should this be manually tested?
```
Run the following commands in your terminal after cloning the project
1. cd ah-92explorers-frontend
2. git checkout ft-fetch-article-feed-168066525
3. `npm i` to install all dependencies 
4. `npm run dev` and navigate to localhost:8080
```
#### What are the relevant pivotal tracker stories?
[#168066525](https://www.pivotaltracker.com/story/show/168066525)
#### Screenshots (if appropriate)
Screen Shot 2019-09-03 at 14 46 56
#### Checklist

- [x] My code follows the style guidelines of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [x] I don't have more than 3 major commits on this PR

### CODE REVIEW

File: .codeclimate.yml [hunk 1]
```diff
@@ -22,4 +21,4 @@ checks:
   similar-code:
     enabled: false
   identical-code:
-    enabled: false
+    enabled: false
```
Comment (elemanhillary-zz) [inline .codeclimate.yml]: insert anew line at EOF, please

**Review by elemanhillary-zz — CHANGES_REQUESTED** [review]
Fix all files that need a new line at the EOF, please

**Review by elemanhillary-zz — DISMISSED** [review]
The feature is incomplete in the following ways:
- The homepage does not consist of articles, with there article clickable links
- The homepage is unresponsive
- Pagination does not exist
- And more....
Reach me for help we do this.

**Review by Hervera — DISMISSED** [review]
Remove the merge conflicts

**Review by elemanhillary-zz — DISMISSED** [review]
kindly implement the responsiveness on the page and let article clicks on images

**Review by niyoceles — DISMISSED** [review]
Kindly remove all unused buttons  and add the link of sign up button

**Review by Hervera — DISMISSED** [review]
Ready to be merged

**Review by niyoceles — DISMISSED** [review]
Feedback implemented well

**Review by Hervera — APPROVED** [review]
Ready to be merged

**Review by Hervera — CHANGES_REQUESTED** [review]
Squash the commits

**Review by elemanhillary-zz — DISMISSED** [review]
ready for merge

**Review by elemanhillary-zz — DISMISSED** [review]
ready for merge

**Review by Hervera — DISMISSED** [review]
Ready to be merged

**Review by niyoceles — DISMISSED** [review]
ready to be merged.

**Review by Hervera — APPROVED** [review]
Ready to be merged

**Review by niyoceles — APPROVED** [review]
ready for merged.

**Review by elemanhillary-zz — APPROVED** [review]
ready for merge

**Pull request merged**